s4e-backend/src/main/java/pl/cyfronet/s4e/bean/Group.java

```diff
@@ -0,0 +1,44 @@
+package pl.cyfronet.s4e.bean;
+
+import lombok.Builder;
+import lombok.Data;
+import org.hibernate.annotations.NaturalId;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotEmpty;
+import java.util.List;
+
+@Entity
+@Data
+@Builder
+@Table(uniqueConstraints = @UniqueConstraint(columnNames = {"name", "slug", "institution_id"}))
```

Comment:
Shouldn't have unique separately for (name, inst_id) and (slug, inst_id)?

Reply:
nope, but now is also bad, slag should be unique, and constraint for name-institution_id :)

Reply:
Yeah, you're right